orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -969,3 +970,26 @@ def softmax(g, input, dim, dtype=None):
     softmax = g.op("Softmax", casted_input, axis_i=dim)
 
     return softmax
+
+
+ATEN_SDPA_FALLBACK = os.getenv("ORTMODULE_ATEN_SDPA_FALLBACK", None)
+if ATEN_SDPA_FALLBACK:
+    # based on the following internal PyTorch kernel for efficient attention:
+    # https://github.com/pytorch/pytorch/blob/c12a4f2e65ad41b739aab1a261e2336b4a79fcfb/aten/src/ATen/native/native_functions.yaml#L14778
+    @register_symbolic("scaled_dot_product_attention")
```

Comment:
if the backward op we override is  "_scaled_dot_product_efficient_attention", I think the correct foward op we should override is "_scaled_dot_product_efficient_attention", otherwise, the fw and bw might NOT be the right pair.

Reply:
The nomenclature used here matches that used in [symbolic_opset14.py](https://github.com/pytorch/pytorch/blob/ef0511245a92bae7057c195dcae2efc237b96f16/torch/onnx/symbolic_opset14.py#L140). For this reason, I believe scaled_dot_product_attention is correct.

Reply:
I see. If so, I think the condition we can enable the fallback is: the selected kernel is _scaled_dot_product_efficient_attention not others, otherwise the fw and bw pair is not correct. Could you pls add some notes in the env var description. I don't know how to check the selected kernel though, maybe you have some suggestions for users to check.

Reply:
Added some more docs.